web/modules/custom/server_style_guide/src/Controller/StyleGuideController.php

```diff
@@ -0,0 +1,320 @@
+<?php
+
+namespace Drupal\server_style_guide\Controller;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Block\BlockManager;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Render\Renderer;
+use Drupal\Core\Url;
+use Drupal\Core\Utility\LinkGenerator;
+use Drupal\server_general\ButtonBuilderTrait;
+use Drupal\server_general\ComponentWrapTrait;
+use Drupal\server_general\TagBuilderTrait;
+use Drupal\taxonomy\Entity\Term;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides route responses for the style-guide module.
+ */
+class StyleGuideController extends ControllerBase {
+
+  use ButtonBuilderTrait;
+  use ComponentWrapTrait;
+  use TagBuilderTrait;
+
+  /**
+   * The block manager service.
+   *
+   * @var \Drupal\Core\Block\BlockManager
+   */
+  private $blockManager;
+
+
+  /**
+   * The renderer service.
+   *
+   * @var \Drupal\server_style_guide\Controller\Renderer
+   */
+  protected $renderer;
+
+  /**
+   * The link generator service.
+   *
+   * @var \Drupal\Core\Utility\LinkGenerator
+   */
+  protected $linkGenerator;
+
+  /**
+   * Class constructor.
+   */
+  public function __construct(BlockManager $block_manager, Renderer $renderer, LinkGenerator $link_generator) {
+    $this->blockManager = $block_manager;
+    $this->renderer = $renderer;
+    $this->linkGenerator = $link_generator;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('plugin.manager.block'),
+      $container->get('renderer'),
+      $container->get('link_generator')
+    );
+  }
+
+  /**
+   * Returns the "Style Guide" page.
+   *
+   * @return array
+   *   A simple renderable array.
+   */
+  public function styleGuidePage() {
+    $card_image = $this->getPlaceholderImage(600, 520);
+
+    $tags = [
+      $this->getMockedTag('The transporter'),
+      $this->getMockedTag('Is more girl'),
+    ];
+
+    $many_tags = $tags + [
+      $this->getMockedTag('The flight'),
+      $this->getMockedTag('bare klingon'),
+      $this->getMockedTag('Dogma doesn’t balanced understand'),
+      $this->getMockedTag('The plank hails with courage'),
+      $this->getMockedTag('burn the freighter until it rises'),
+    ];
+
+    $single_card_simple = [
+      '#theme' => 'server_theme_card__simple',
+      '#image' => $card_image,
+      '#title' => 'The source has extend, but not everyone fears it.',
```

Comment:
```suggestion
      '#title' => $this->t('The source has extend, but not everyone fears it.'),
```

Do we wanna bother translating this gibberish? There are a few more missed ones below. I won't highlight them all.

Reply:
Neh, I think we can avoid the `$this->t()`